packages/odyssey-react-mui/src/@types/i18next.d.ts

```diff
@@ -18,5 +18,6 @@ declare module "i18next" {
     resources: {
       translations: typeof en;
     };
+    returnNull: false;
```

Comment:
The reason we need to do this is that otherwise function calls of the translate method throw an error because of the possibility of a null value being return. The solution implemented is detailed [here](https://www.i18next.com/overview/typescript#argument-of-type-defaulttfuncreturn-is-not-assignable-to-parameter-of-type-xyz)